tapdb/universe_forest.go

```diff
@@ -131,6 +131,33 @@ func (b *BaseUniverseForest) RootNodes(
 	return uniRoots, nil
 }
 
+// FetchIssuanceProof returns an issuance proof for the target key. If the key
+// doesn't have a script key specified, then all the proofs for the minting
+// outpoint will be returned. If neither are specified, then proofs for all the
+// inserted leaves will be returned.
+func (b *BaseUniverseForest) FetchIssuanceProof(ctx context.Context,
+	id universe.Identifier,
+	universeKey universe.BaseKey) ([]*universe.IssuanceProof, error) {
+
+	var (
+		readTx = NewBaseUniverseReadTx()
+		proofs []*universe.IssuanceProof
+	)
+
+	dbErr := b.db.ExecTx(ctx, &readTx, func(dbTx BaseUniverseForestStore) error {
+		var err error
+		proofs, err = universeFetchIssuanceProof(
```

Comment:
See other comment, this also needs to have a multiverse root inclusion proof along side it.